src/helm/benchmark/annotation/wildbench_annotator.py

```diff
@@ -1,9 +1,11 @@
 import re
 from typing import Any
 from importlib.resources import files
+from typing import Dict
 
 from helm.benchmark.adaptation.request_state import RequestState
 from helm.benchmark.annotation.annotator import Annotator
+from helm.benchmark.annotation.model_as_judge import _AnnotatorModelInfo
```

Comment:
@yifanmai Should we rename the `_AnnotatorModelInfo` class and remove the leading underscore?

Reply:
Yes, let's do that.